src/exasolvs/queryrenderer/ImportAppender.lua

```diff
@@ -0,0 +1,80 @@
+local AbstractQueryRenderer = require("exasolvs.queryrenderer.AbstractQueryAppender")
+local SelectAppender = require("exasolvs.queryrenderer.SelectAppender")
+local Query = require("exasolvs.Query")
+
+--- Appender that can add top-level elements of a `SELECT` statement (or sub-select).
+-- @classmod ImportAppender
+local ImportAppender = {}
+ImportAppender.__index = ImportAppender
+setmetatable(ImportAppender, {__index = AbstractQueryRenderer})
+
+--- Create a new query renderer.
+-- @param out_query query structure as provided through the Virtual Schema API
+-- @return query renderer instance
+function ImportAppender:new(out_query)
+    local instance = setmetatable({}, self)
+    instance:_init(out_query)
+    return instance
+end
+
+function ImportAppender:_init(out_query)
+    AbstractQueryRenderer._init(self, out_query)
+end
+
+function ImportAppender:_append_select_list_elements(select_list)
+    for i = 1, #select_list do
+        local element = select_list[i]
+        self:_comma(i)
+        self:_append_expression(element)
+    end
+end
+
+function ImportAppender:_append_connection(connection)
+    self:_append(' FROM EXA AT "')
+    self:_append(connection)
+    self:_append('"')
+end
+
+--- Get the statement with extra-quotes where necessary as it will be embedded into the IMPORT statement.
+-- @return statement with escaped single quotes
```

Comment:
```suggestion
--- Get the statement with extra-quotes where necessary as it will be embedded into the IMPORT statement.
-- @param statement statement for which to escape quotes
-- @return statement with escaped single quotes
```